php-frontend/src/test/java/org/sonar/php/parser/expression/NamespaceNameTest.java

```diff
@@ -36,8 +36,9 @@ public void test() {
       .matches("namespace\\NS")
       .matches("namespace\\NS1\\NS2\\Name")
 
```

Comment:
I would explicitly add here the examples with `namespace if` and `namespace for\while`.

Reply:
This would be the correct spot. `namespace if` is a `NamespaceStatement` and not a `NamespaceName`. However, I will add a test to `NamespaceStatementTest`